src/Microsoft.AspNetCore.Mvc.Testing/Internal/TestCulture.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Testing.Internal
+{
+    public class TestCulture
+    {
+        public string Culture { get; set; }
```

Comment:
I find easier to just use a string here and do new CultureInfo(culture) ourselves. It's less verbose and the user still needs to know the culture string for whatever culture they want.